README.md

```diff
@@ -2,67 +2,94 @@
 control through an API with bindings in Tcl and Python.
 
-OpenROAD is a foundational building block in open-source digital flows
-such as [OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts),
+OpenROAD is used in both open-source and commercial IC design
+flows such as native [OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts),
 [OpenLane](https://github.com/The-OpenROAD-Project/OpenLane) from
 [Efabless](https://efabless.com/), [Silicon Compiler](https://github.com/siliconcompiler/siliconcompiler)
 from Zero ASIC, [Hammer](https://docs.hammer-eda.org/en/latest/Examples/openroad-nangate45.html)
 from Berkeley and [OpenFASoC](https://github.com/idea-fasoc/OpenFASOC) for
 mixed-signal design flows.
 
-OpenROAD users span hardware designers, industry collaborators,
-enthusiasts, academics, and researchers.
-
-Two main flow controllers are supported by the
-[OpenROAD](https://github.com/The-OpenROAD-Project/OpenROAD)
-project repository:
-
--   [OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts) -
-     Supported by the OpenROAD project
-
--   [OpenLane](https://github.com/The-OpenROAD-Project/OpenLane) -
-     Supported by [Efabless](https://efabless.com/)
-
-The OpenROAD flow delivers an autonomous, no-human-in-the-loop, 24 hour
-turnaround from RTL to GDSII for design exploration and physical design
-implementation.
-
-![rtl2gds.webp](./docs/images/rtl2gds.webp)
+OpenROAD fosters a vibrant ecosystem of users through active
+collaboration and partnership through software development and key
+alliances.  Our growing user community includes hardware designers,
+software engineers, industry collaborators, VLSI enthusiasts,
+students and researchers.
+
+OpenROAD strongly advocates and enables IC design-based education
+and workforce development initiatives through training content and
+courses across several global universities, the Google-SkyWater
+[shuttles](https://platform.efabless.com/projects/public), design
+contests and IC design workshops. The OpenROAD flow has been
+successfully used to-date, in over 600, silicon-ready tapeouts for
+technologies up to 12nm.
+
+## OpenROAD Flow Stages
+OpenROAD includes a native flow - [OpenROAD-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts).
+However, it also enables the creation of any custom flow controllers
+based on the underlying tools, database and analysis engines.
+
+OpenROAD-flow-scripts(ORFS) is a fully autonomous, RTL-GDSII flow
+for rapid architecture and design space exploration, early prediction
+of QoR and detailed physical design implementation. However, ORFS
+also enables manual intervention for finer user control of individual
+flow stages through tcl commands and python APIs.
+
+Figure below shows the main stages of the OpenROAD-flow-scripts:
+
+![ORFS_Flow.webp](./docs/images/ORFS_Flow.webp)
```

Comment:
I am usually not a fan of diagrams in an image format. I prefer SVG because

1. They are usually smaller file size
2. Images cannot be edited easily, whereas SVG can (SVG is both the source and the output).
3. They do not have compression and risk unreadable text.

If you can upload an SVG, it will make maintaining this diagram easier in the future.
Otherwise:
* OpenRoad --> OpenROAD
* flow script --> Flow Scripts
* Tool: Yosys is in a different color than the rest
* Input file --> Input files
* Output file --> Output files
* We typically use Floorplan as one word
* all necessary file --> all necessary files
* There's a lot of unnecessary capitalization used - please proofread
* KLayout isn't mentioned but Yosys is? There is mixed use of tool mention or not mentioned. Please make uniform.

Reply:
This flow chart created using Xmind and converted to PNG. Let me check for svg options.

Reply:
Updated the changes. I have backup of  `.xmind` of this flow chart. We can modify/update in future if needed. @rovinski

Reply:
Were you unable to generate an SVG?

Reply:
Your copy of the `.xmind` could easily be lost because it isn't checked in the repo...

Reply:
Xmind pro version has svg conversion. Free version doesn't. I will check with vitor on how keep backup of .Xmind file for future updates.

Reply:
I like [Inkscape](https://inkscape.org/) or [draw.io](https://draw.io) which are both completely free.

Reply:
@rovinski @vvbandeira 
Please check the updated `.svg` file and let me know, if need any further updates.